pipelines/base/config-dump.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: dump-config
+spec:
+  params:
+  - name: REPO_NAME
+    type: string
+    default: .github
+  - name: BRANCH_NAME
+    type: string
+    default: add-initial-peribolos-config
+  - name: BASE_BRANCH
+    type: string
+    default: main
+  - name: GIT_EMAIL
+    type: string
+  - name: GIT_NAME
+    type: string
+  workspaces:
+  - name: secret-data
+    description: location for oauth token and ssh config
+  steps:
+    - name: dump-and-push-config
+      image: quay.io/operate-first/opf-toolbox
+      script: |
+        #!/bin/bash
+
+        TOKEN=$( cat $(workspaces.secret-data.path)/token )
+        ORG_NAME=$( cat $(workspaces.secret-data.path)/orgName )
+
+        git config --global user.email "$(params.GIT_EMAIL)"
+        git config --global user.name "$(params.GIT_NAME)"
```

Comment:
We should pass these values from the controller, that is correct. Just for the feature reference, user name and email should follow https://github.community/t/logging-into-git-as-a-github-app/115916